src/main/java/uk/gov/hmcts/darts/arm/rpo/impl/ArmRpoApiImpl.java

```diff
@@ -33,6 +36,9 @@ public class ArmRpoApiImpl implements ArmRpoApi {
     private final ArmRpoClient armRpoClient;
     private final ArmRpoService armRpoService;
 
+    @Value("${darts.storage.arm-api.arm-service-entitlement}")
```

Comment:
This attribute is included in the ArmApiConfigurationProperties which I have included in my PR for getStorageAccounts, which is just waiting for a second review to get merged.